### PR TITLE
Added Admin Router service endpoint documentation.

### PR DESCRIPTION
**Pull request by joerg84**
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [x] High
- [ ] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

**Review by vespian — APPROVED** [review]
🚢

File: 1.8/development/dcos-integration.md [hunk 1]
```diff
@@ -12,17 +12,28 @@ When a DC/OS Service is installed and ran on DC/OS, the service is generally dep
 
 Admin Router currently supports only one reverse proxy destination.
 
-In order to allow Admin Router to function as a reverse proxy for the DC/OS Service one of the following methods should be used
+## Service Endpoints
 
-## Framework Web UI URL
+Admin Router allows marathon tasks to define custom service UI and HTTP endpoints, which are made available as `/services/<service-name>`. This can be achieved by setting the following marathon task labels:
```
Comment (jgehrcke) [inline 1.8/development/dcos-integration.md]: There is an `s` too much here, it's `/service/..`

File: 1.8/development/dcos-integration.md [hunk 2]
```diff
@@ -12,17 +12,28 @@ When a DC/OS Service is installed and ran on DC/OS, the service is generally dep
...
 
-If the DC/OS Service will register with Mesos as a framework, the `webui_url` framework property may be specified and will be used by Admin Router to proxy requests to the service.
+```
+"labels": {
+    "DCOS_SERVICE_NAME": "service-name",
```
Comment (jgehrcke) [inline 1.8/development/dcos-integration.md]: If we say `<service-name>` (literal) in the text above, then we should also use `<service-name>"` here.

File: 1.8/development/dcos-integration.md [hunk 3]
```diff
@@ -12,17 +12,28 @@ When a DC/OS Service is installed and ran on DC/OS, the service is generally dep
...
+    "DCOS_SERVICE_PORT_INDEX": "0",
+    "DCOS_SERVICE_SCHEME": "http"
+  }
+```
 
-* The URL must NOT end with a backslash (/). For example, this is good `internal.dcos.host.name:10000`, and this is bad `internal.dcos.host.name:10000/`.
+In this case `http://<dcos-cluster>/services/service-name` would be forwarded to the host running the task using the first port allocated to the task.
```
Comment (jgehrcke) [inline 1.8/development/dcos-integration.md]: `/service/` here, too (should probably check the entire doc).

File: 1.8/development/dcos-integration.md [hunk 4]
```diff
@@ -12,17 +12,28 @@ When a DC/OS Service is installed and ran on DC/OS, the service is generally dep
...
+
+In order for the forwarding to work reliably across task failures, we recommend co-locating the endpoints with the task. This way, if the task is restarted on a potentially other host and with different ports, Admin Router will pick up the new labels and update the routing. NOTE: Due to caching there might be an up to 30-second delay until the new routing is working.
+
+We would recommend having only a single task setting these labels for a given service-name.
+In the case of multiple task instances with the same service-name label, admin router will pick one of the tasks instances deterministically, but this might make debugging issues more difficult.
```
Comment (jgehrcke) [inline 1.8/development/dcos-integration.md]: "task instances" or "task's instances"?

File: 1.8/development/dcos-integration.md [hunk 5]
```diff
@@ -12,17 +12,28 @@ When a DC/OS Service is installed and ran on DC/OS, the service is generally dep
...
 
 Since the paths to resources for clients connecting to Admin Router will differ from those paths the Service actually has, ensure the Service is friendly to running behind a proxy. This often means relative paths are preferred to absolute paths. In particular, resources expected to be used by a UI should be verified to work through a proxy.
 
-When the `webui_url` is provided, a link to the service will be available from the DC/OS UI. The link will be generated based on a convention of: `/service/<service_name>`. For example, `<dcos_host>/service/unicorn` is the proxy to the `webui_url`.
+Tasks running in nested [marathon app groups](https://mesosphere.github.io/marathon/docs/application-groups.html) will be available only using their service name (i.e, `/services/<service-name>`) and not considering the marathon app group name (i.e., `/services/app-group/<service-name>`).
```
Comment (jgehrcke) [inline 1.8/development/dcos-integration.md]: Dot (`.`) missing here after the first "i.e".

**Comment by jgehrcke** [comment]
Huhu, and huge hanks! I have left a few comments, and some of them probably apply to a few more places than I have commented on. After taking care of them: :ship:

File: 1.8/development/dcos-integration.md (hunk 1 above)
Comment (sascala) [inline 1.8/development/dcos-integration.md]: 👍

File: 1.8/development/dcos-integration.md (hunk 2 above)
Comment (sascala) [inline 1.8/development/dcos-integration.md]: 👍

File: 1.8/development/dcos-integration.md (hunk 3 above)
Comment (sascala) [inline 1.8/development/dcos-integration.md]: 👍

File: 1.8/development/dcos-integration.md (hunk 4 above)
Comment (sascala) [inline 1.8/development/dcos-integration.md]: the former, AFAICT

Comment (sascala) [inline 1.8/development/dcos-integration.md]: the former

File: 1.8/development/dcos-integration.md (hunk 5 above)
Comment (sascala) [inline 1.8/development/dcos-integration.md]: 👍

**Comment by joerg84** [comment]
@jgehrcke Thanks for your feedback
@sascala Thanks for addressing that feedback directly

From my side: LGTM

**Comment by sascala** [comment]
Yes, these changes should also apply to 1.9, @joerg84

**Pull request merged**